### ISSUE TITLE
[Docs]: Web documentation of CLI options should clearly indicate `--` prefix

### ISSUE THREAD

**Issue by polarathene** [smallstep/cli#359]
I've used a few Go based CLI utilities where long option names only use a single `-` and [viewing the docs for `step`](https://smallstep.com/docs/cli/ca/init/) this seemed like it was probably the case as well. Trying that out however was rejected and the CLI output a list of valid options each prefixed with `--`, the character used on the docs doesn't appear to be a single `-`, it's a bit longer so perhaps it was `--` in whatever source document it was written in, but was then converted to something like an em-dash?

## TL;DR

It would be helpful if the web docs presented two distinct `-`as the option prefix (eg `--pki`), it presently looks like `-pki`.

**Comment by polarathene** [comment]
Nope, nevermind this was a mistake on my part, single dash works fine.

Error I was getting was `flag provided but not defined: -password`, which isn't an option, I was after `-password-file`.
